### DISCUSSION TITLE
2.0版本页面评论模块有时加载不出来

### DISCUSSION THREAD

**Discussion by inmarkd** [MaLuns/hexo-theme-async#61] [Q&A 疑难杂症] [answered]
- 今天更新了2.0版 发现评论模块有时加载不出来，需要刷新才能二次加载
- 代码块复制失败。

**Comment by MaLuns** [comment, marked as answer]
主题里 twikoo 初始化地方问题，可以先在配置里将 `swup: false`，这个会在下个版本修复。
``` js
const twikooConfig = <%- JSON.stringify(theme.comment.twikoo) %>;

你也可以临时将上面的调整为

var twikooConfig = <%- JSON.stringify(theme.comment.twikoo) %>;
```
复制问题，看你站点时 http 的，浏览器默认是禁止的

![image](https://user-images.githubusercontent.com/31614024/216045716-2f6d7e29-0409-4752-ae80-21bed32abef3.png)

**Reply by inmarkd** [reply]
好的 感谢答复。
